tests/mgbench/validation.py

```diff
@@ -88,6 +88,13 @@ def pars_args():
         help="number of workers used to import the dataset",
     )
 
+    parser.add_argument(
```

Comment:
Just a comment here for understanding so this script was added so we can check if Neo4j and Memgraph return the same results from `LDBC` queries, since those queries are complicated. 

It is sufficient to run just a single query and get the results back from the query, there is no need for concurrency, hence multiple workers. 

I referred to `validated.py` as being out of date because it didn't fill the purpose we were discussing  (the persistent results between runs, etc). Also,  not run in our CI/CD environment for things you mentioned you need (validating results). 

There is a lot of duplication and hardcoding here in this script.

Reply:
BenchmarkContext object was constructed without providing number of workers for benchmark, it will be overwritten later in client.cpp anyway, but I will remove this to make code more clear.